mavros/include/mavros/mavros_uas.h

```diff
@@ -416,35 +416,45 @@ class UAS {
 	}
 
 	/**
+	 * @brief Frame transform options when applying rotations to data
+	 */
+	enum TRANSFORM_TYPE{
+		PLATFORM_TO_ENU,
+		ENU_TO_PLATFORM,
+		NED_TO_ENU,
+		ENU_TO_NED
+	};
+
+	/**
 	 * @brief Transform frame between ROS and FCU. (Vector3d)
 	 *
 	 * General function. Please use specialized enu-ned and ned-enu variants.
 	 */
-	static Eigen::Vector3d transform_frame(const Eigen::Vector3d &vec);
+	static Eigen::Vector3d transform_frame(const Eigen::Vector3d &vec, const TRANSFORM_TYPE &transform);
```

Comment:
"By reference" not needed for small types like enum: `const TransformType type`.